Library/Formula/tcc.rb

```diff
@@ -5,11 +5,17 @@ class Tcc < Formula
   url 'http://download.savannah.gnu.org/releases/tinycc/tcc-0.9.26.tar.bz2'
   sha1 '7110354d3637d0e05f43a006364c897248aed5d0'
 
+  option "with-cross", "Build cross compilers"
```

Comment:
For anything it targets, as listed in the docs. Listing them all would be odd, as some of them are experimental. LLVM just has an option "Build all target backends"